emily/handler/src/database/accessors.rs

```diff
@@ -838,7 +844,9 @@ pub async fn get_limits(context: &EmilyContext) -> Result<Limits, Error> {
         global_cap.rolling_withdrawal_blocks,
         global_cap.rolling_withdrawal_cap,
     )
-    .await?;
+    .await
+    .ok()
+    .flatten();
```

Comment:
This will convert any error to None and will allow to return other limits

Reply:
Could we add an `inspect_err` so that it's at least logged?

Reply:
Oh, I guess you do that above. But you could explicitly log here that you're returning `None` from get limits due to chainstate failure.

Reply:
We need to check how the bridge handles `None` as well.

Reply:
Yeah, `inspect_err` worth adding. However, I suspect it will be always `Error::NotFound`

Reply:
In general I'd suggest instead of treating `None` as no limit here and in other limits as well, treat `None` as error, and `Some(u64::MAX)` as no limit

Reply:
Nice, closing this now based on standup discussion 👍 The bridge will effectively fall-back to the per-withdrawal limits with this, and the signers will do their own thing (calculations).